content/index.ko.md

```diff
@@ -22,7 +23,7 @@ github:
 # Translate the Working Group name. Leave the Working Group acronym in English.
 # Do not change the dates in the footer below.
 footer: >
-  <p><strong>날짜:</strong> 2019년 7월 11일 업데이트됨. 2005년 2월 처음 발행됨. CHANGELOG.</p>
+  <p><strong>날짜:</strong> 2024년 2월 9일 업데이트됨. 2005년 2월 처음 발행됨. CHANGELOG.</p>
```

Comment:
@hwahyeon Please keep the date from the original document: 20 November 2023. This date reflects the last updated date of the original English document.